### PR TITLE
add tab form

### PR DESCRIPTION
**Pull request by piensaenpixel**
@javierarce could you take a look?
image

### CODE REVIEW

**Comment by piensaenpixel** [comment]
ping @javierarce

File: src/scss/cdb-components/shapes/paragraph.scss [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      <span class="CDB-Shape-paragraphBar CDB-Shape-paragraphBar--04"></span>
+    </li>
+  </ul>
+</div>
+<br/>
+<br/>
+<br/>
+<br/>
+<div class="CDB-Shape">
+  <ul class="CDB-Shape-paragraph is-left is-active">
```
Comment (javierarce) [inline src/scss/cdb-components/shapes/paragraph.scss]: `.is-left` is not defined, right? I think we could either remove it or define it next to `.is-center`, `.is-right`.

**Comment by javierarce** [comment]
Except for a comment I added, the rest looks 👍

File: src/scss/cdb-components/shapes/paragraph.scss (hunk 1 above)
Comment (piensaenpixel) [inline src/scss/cdb-components/shapes/paragraph.scss]: right :D thanks!

**Pull request merged**